tools/pontoon/translate.md

```diff
@@ -0,0 +1,105 @@
+
+![Hovering over the string will highlight in sidebar](/assets/images/pontoon/translate/hover_incontext.png)
+
+Once you double click a string, the sidebar will display the editing space.
+
+![Translate in-context projects in the sidebar](/assets/images/pontoon/translate/translate_in_sidebar.png)
+
+Input your translation or review the suggestion and click `SAVE` or `SUGGEST`. Once your translation is saved, the editing space will move to the next string, and that string will be highlighted in the in-context view.
+
+To return to the list of strings in the sidebar, click `BACK TO LIST` at the top-left of the sidebar.
+
+Interactive website features will work in the in-context view. Some strings that you can see in the sidebar will not be visible in the in-context view until you activate the website feature. In the picture below, the translator has clicked on the hamburger to reveal a drop-down menu.
+
+![Interactive Hamburger menu](/assets/images/pontoon/translate/interactive_features.png)
+
+## Performing batch actions on strings
+
+Translators can perform mass actions on batches of strings by clicking on the square to the left of any string in the sidebar. When clicked, the square will show a checkmark and the editing space will show the mass action window.
+
+![Mass action window](/assets/images/pontoon/translate/mass_action.png)
+
+At the top-left of the mass action window, the translator can return to the editing space by clicking on the `x` icon. To the right of the exit icon, the number of strings currently selected is displayed. At the top-right of the mass action window, the translator can select all of the strings in the sidebar.
+
+In the `REVIEW TRANSLATIONS` section, the translator can approve suggestions or remove approved translations for all selected strings. Upon clicking `APPROVE ALL`, the text in the green button will be replaced with the number of new approved translations.
+
+In the `REPLACE IN TRANSLATIONS` section, the translator can input a series of characters to search for, and a series of characters to use as a replacement. This is a basic find and replace feature that will work only on the selected strings.
+
+Note that mass actions are only available to users with [translators rights](users.md#user-roles).
+
+## Downloading and uploading translations
+
+Pontoon features the ability to download and upload translations, including translation memories. To access these features, click on your user icon in the top-right section of Pontoon. Note that you must be in the translation workspace for the download/upload options to display in the dropdown menu.
+
+![Profile dropdown menu](/assets/images/pontoon/translate/profile_menu.png)
+
+Only translators can download a translation memory (.tmx) and translations, or upload translations. When downloading translations, the resource currently selected is downloaded in its original format, it’s not converted in any way.
```

Comment:
Everybody can download. Translators can upload.

If the project contains more than 10 files, you download file-by-file. For other projects a zip of all files is downloaded. (The reason for this inconsistency is performance).